__tests__/components/__snapshots__/RadioButtonGroup.spec.js.snap

```diff
@@ -143,7 +143,7 @@ exports[`RadioButtonGroup renders correctly 1`] = `
 
 .c4:hover input:not([disabled]) + div,
 .c4:hover input:not([disabled]) + span {
-  border-color: #527699;
+  border-color: #D1D8F2;
```

Comment:
Why did the radio get affected?

Reply:
That's very strange indeed. taking a look